### ISSUE TITLE
unable to get example Ollama script to work on my machine

### ISSUE THREAD

**Issue by awdemos** [qdrant/landing_page#1263]
On this page I was not able to get the script to run https://qdrant.tech/documentation/embeddings/ollama/ 

```
import qdrant_client
from qdrant_client.models import Batch
from ollama import Ollama

# Initialize Ollama model
model = Ollama("ollama-unique")

# Generate embeddings for niche applications
text = "Ollama excels in niche applications with specific embeddings."
embeddings = model.embed(text)

# Initialize Qdrant client
qdrant_client = qdrant_client.QdrantClient(host="localhost", port=6333)

# Upsert the embedding into Qdrant
qdrant_client.upsert(
    collection_name="NicheApplications",
    points=Batch(
        ids=[1],
        vectors=[embeddings],
    )
)
```
In particular python was complaining about ```from ollama import Ollama```.
I had to make this script to work:

```
from qdrant_client import QdrantClient
from qdrant_client.models import Distance, VectorParams, CollectionConfig
import ollama

# Initialize Ollama client
client = ollama.Client(host="localhost")

# Initialize Qdrant client
qdrant_client = QdrantClient(host="localhost", port=6333)

# Text to embed
text = "Ollama excels in niche applications with specific embeddings"

# Generate embeddings
response = client.embeddings(model="llama3.2", prompt=text)
embeddings = response['embedding']  # Note: it's 'embedding' not 'embeddings'

# First, create the collection if it doesn't exist
try:
    qdrant_client.create_collection(
        collection_name="NicheApplications",
        vectors_config=VectorParams(size=len(embeddings), distance=Distance.COSINE)
    )
except Exception as e:
    print(f"Collection might already exist: {e}")

# Then upsert the embeddings
qdrant_client.upsert(
    collection_name="NicheApplications",
    points=[{
        "id": 1,
        "vector": embeddings,
        "payload": {"text": text}
    }]
)
```

**Comment by Anush008** [comment]
Hey @awdemos.
I added the fixes and some general improvements in #1264.
Could you please review it?

**Comment by Anush008** [comment]
Hey @awdemos.
Thanks for raising the issue.
I added the fixes and some general improvements in #1264. Could you please review it?
